src/content/whats-new/2024/06/whats-new-06-11-click-to-parse.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Streamline log parsing'
+summary: 'Use our new options to generate ingest parsing or query time parsing rules'
+releaseDate: '2024-06-11'
+learnMoreLink: 'https://newrelic.com/blog/how-to-relic/click-to-parse'
+getStartedLink: 'https://youtu.be/tvK6MlkvD6Y?si=5klygPBzQyY5qsI8'
+---
+
+While the majority of valuable parsing information is automatically gathered during the ingestion of logs into the New Relic platform, building additional queries can be labor-intensive. Logs parsing just became easier for both types of parsing at New Relic: 
+
+* Ingest parsing: These are rules that create permanent changes in your logs and are based on Grok rules. In the past, you had to create your own Grok rules, but now we can create these complex rules for you with our NRAI assistant.  
```

Comment:
```suggestion
* Ingest parsing: These are rules that create permanent changes in your logs and are based on New Relic AI  rules. In the past, you had to create your own New Relic AI  rules, but now we can create these complex rules for you with our New Relic AI assistant.  
```

Reply:
I don't believe we use NRAI as a shorthand, but I could be wrong. I would check out our usage dictionary to be sure. If we do allow NRAI, I would:

1. Use New Relic AI (NRAI) on the first instance
2. Update all subsequent references to New Relic AI/Grok to say NRAI

Reply:
This is actually meant to be Grok!